python/ql/test/query-tests/Security/lib/lxml/etree/__init__.py

```diff
@@ -0,0 +1,25 @@
+
+class etree(object):
+    # pass
+    def parse(source, parser=None, base_url=None):
+        return ElementTree()
+
+
+class ElementTree(object):
+    def xpath(self, _path, namespaces=None, extensions=None, smart_strings=True, **_variables):
+        pass
```

Comment:
Do you need this for anything? If not, can we please remove it? If you do need it, it should be changed since `type(etree.ElementTree)` gives `cython_function_or_method` -- certainly not a class :wink:

Reply:
I have changed it to `_ElementTree` now the types should match.